simulator/scheduler/plugin/plugins.go

```diff
@@ -257,16 +274,29 @@ func mergePluginSet(inTreePluginSet, outOfTreePluginSet v1beta2.PluginSet) v1bet
 	return v1beta2.PluginSet{Enabled: enabledPlugins}
 }
 
-// registeredFilterScorePlugins returns all registered score plugin and filter plugin.
-func registeredFilterScorePlugins() ([]v1beta2.Plugin, error) {
+// registeredFilterPostFilterScorePlugins returns all registered score plugin and filter plugin.
+func registeredFilterPostFilterScorePlugins() ([]v1beta2.Plugin, error) {
 	registeredfilterpls, err := config.RegisteredFilterPlugins()
 	if err != nil {
 		return nil, xerrors.Errorf("get registered filter plugins: %w", err)
 	}
+	registerdpostfilterpls, err := config.RegisteredPostFilterPlugins()
+	if err != nil {
+		return nil, xerrors.Errorf("get registered postFilter plugins: %w", err)
+	}
 	registeredscorepls, err := config.RegisteredScorePlugins()
 	if err != nil {
 		return nil, xerrors.Errorf("get registered score plugins: %w", err)
 	}
+	plgs := [][]v1beta2.Plugin{
+		registeredscorepls,
+		registerdpostfilterpls,
+		registeredfilterpls,
+	}
+	result := make([]v1beta2.Plugin, 0, len(plgs))
+	for _, p := range plgs {
+		result = append(result, p...)
+	}
 
-	return append(registeredscorepls, registeredfilterpls...), nil
+	return result, nil
```

Comment:
You don't need to calculate result by `for` loop. Maybe like `return append(append(registeredscorepls, registeredfilterpls...), registerdpostfilterpls...), nil` seems enough